Console/Command/ResetCommand.php

```diff
@@ -108,7 +108,7 @@ private function resetConfig()
     {
         $this->resourceConnection->getConnection()->delete(
             $this->resourceConnection->getTableName('core_config_data'),
-            "path LIKE '" . \Yotpo\Yotpo\Helper\Data::XML_PATH_ALL . "/%'"
+            "path LIKE '" . \Yotpo\Yotpo\Model\Config::XML_PATH_YOTPO_ALL . "/%'"
```

Comment:
what is the reason for this change? what is the connection to single store mode?

Reply:
It's a bug that I found during the single-store issue (actually I found 2 bugs & solved them on a different commit).